app/src/main/java/org/oppia/android/app/topic/TopicTab.kt

```diff
@@ -44,20 +54,38 @@ enum class TopicTab(
   companion object {
     /**
      * Returns the [TopicTab] corresponding to the specified tab position, considering whether the
-     * info and practice tabs are enabled per [enableExtraTopicTabsUi].
+     * info and practice tabs are enabled per [enableTopicInfoTab] and [enableTopicPracticeTab]
+     * respectively.
      */
-    fun getTabForPosition(position: Int, enableExtraTopicTabsUi: Boolean): TopicTab {
+    fun getTabForPosition(
+      position: Int,
+      enableTopicInfoTab: Boolean,
+      enableTopicPracticeTab: Boolean
+    ): TopicTab {
       return checkNotNull(
         values().find {
-          position == if (enableExtraTopicTabsUi) {
+          position == if (enableTopicInfoTab && enableTopicPracticeTab) {
             it.positionWithFourTabs
-          } else it.positionWithTwoTabs
+          } else if (enableTopicInfoTab) {
+            it.positionWithThreeTabsWithInfo
+          } else if (enableTopicPracticeTab) {
+            it.positionWithThreeTabsWithPractice
+          } else {
+            it.positionWithTwoTabs
+          }
```

Comment:
Based on my suggestion ablove, this could be condensed into:
```suggestion
       fun getTabForPosition(position: Int, enableInfo: Boolean, enablePractice: Boolean): TopicTab {
      return values().find { it.getPosition(enableInfo, enablePractice) == position }
        ?: error("No tab corresponding to position: $position")
    }

Reply:
Done.